tx-helper.go

```diff
@@ -0,0 +1,150 @@
+package ledger
+
+import (
+	"crypto/sha512"
+	"encoding/hex"
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/spacemeshos/ed25519"
+)
+
+// Transaction info struct
+type txInfo struct {
+	PublicKey []byte
+	NetworkID []byte
+	Type      byte
+	Nonce     uint64
+	To        []byte
+	GasLimit  uint64
+	GasPrice  uint64
+	Amount    uint64
+	Data      []byte
+}
+
+// Convert uint64 value to byte array
+func uint64ToBuf(value uint64) []byte {
+	data := make([]byte, 8)
+	data[0] = byte((value >> 56) & 0xff)
+	data[1] = byte((value >> 48) & 0xff)
+	data[2] = byte((value >> 40) & 0xff)
+	data[3] = byte((value >> 32) & 0xff)
+	data[4] = byte((value >> 24) & 0xff)
+	data[5] = byte((value >> 16) & 0xff)
+	data[6] = byte((value >> 8) & 0xff)
+	data[7] = byte((value) & 0xff)
+	return data
+}
+
+// Load transactoin info from JSON file
+func loadTxInfo(fileName string) (*txInfo, error) {
+	txInfo := &txInfo{}
+	jsonFile, err := os.Open("./test" + string(os.PathSeparator) + fileName)
+	if err != nil {
+		return nil, err
+	}
+	defer jsonFile.Close()
+
+	byteValue, err := ioutil.ReadAll(jsonFile)
+	if err != nil {
+		return nil, err
+	}
+
+	var data map[string]interface{}
+	json.Unmarshal([]byte(byteValue), &data)
```

Comment:
Please handle the error here.

Also (you're converting to-from the same type):
```suggestion
	json.Unmarshal(byteValue, &data)
```

In general, you can unmarshal JSON directly to a struct in Go, but you might encounter some difficulties with hex strings and flattening the `data` field. While this function isn't pretty - maybe it's not worth the effort to improve it - I leave it up to you.